library/src/test/java/com/bumptech/glide/GlideTest.java

```diff
@@ -132,6 +137,7 @@ public Boolean answer(InvocationOnMock invocation) throws Throwable {
   @After
   public void tearDown() {
     Glide.tearDown();
+    Glide.setModulesEnabled(true);
```

Comment:
I meant `Glide.tearDown` not the `@After`, though I guess it doesn't matter much.